client/lib/tree-select/index.js

```diff
@@ -57,6 +75,8 @@ export default function treeSelect( getDependents, selector ) {
  * The last map is a regular one because the the key for the last map is the string results of args.join().
  */
 function insertDependentKey( map, key, currentIndex, arr ) {
+	key = key || STATIC_FALSY_KEY;
```

Comment:
how would you feel about making this a separate variable so we don't mutate the args? I find this often confusing to read code that _does_.

```js
function insertDependentKey( map, rawKey, currentIndex, arr ) {
	const key = rawKey || STATIC_FALSY_KEY;

	…
}
```

Reply:
Introduced a `weakMapKey` variable.